tests/Range.test.ts

```diff
@@ -0,0 +1,94 @@
+import { describe, test, expect } from "vitest";
+import range from "../src";
+
+describe("range", () => {
+    test("get trap", () => {
```

Comment:
A short test description about the main feature this test covers would be good

Reply:
duplicated